src/app/pages/network/components/open-vpn-server-config/open-vpn-server-config.component.ts

```diff
@@ -178,4 +184,21 @@ export class OpenVpnServerConfigComponent implements OnInit {
 
     this.subscriptions.push(topologySubscription);
   }
+
+  unsetCertificates(): void {
+    this.dialogService.confirm({
+      title: this.translate.instant('Warning'),
+      message: this.translate.instant('This operation will unset any certificates assgined to OpenVPN Server configuration. Are you sure you want to proceed?'),
+    }).pipe(
+      filter(Boolean),
+      switchMap(() => {
+        this.appLoaderService.open();
+        return this.ws.call('openvpn.server.update', [{ remove_certificates: true } as OpenvpnServerConfigUpdate]);
```

Comment:
Errors are not handled. If I press the button when no certificates are set, I'll get stuck.